app/helpers/courses_helper.rb

```diff
@@ -11,4 +11,9 @@ def pub_status_str(course)
     when "T" then "Trashed"
     end
   end
+
+  def course_topics(course)
+    topics = course.topics.map(&:title)
```

Comment:
FYI, I support making a wrapper method, rather than having the long pluck/join in the template. Even though it's longer than the single line call (three line method to call a single line), it makes reusability much easier in the future.